apps/web/src/pages/Home/ManagerHomePage/ManagerHomePage.tsx

```diff
@@ -182,38 +182,31 @@ export const Component = () => {
           <CardFlat
             color="tertiary"
             title={intl.formatMessage({
-              defaultMessage: "Gain data insights",
-              id: "3GOVZI",
-              description: "Heading for the direct on digital talent section",
+              defaultMessage: "Hire Indigenous talent",
+              id: "P06ncG",
+              description:
+                "Heading for the IT Apprenticehip program for Indigenous peoples section",
             })}
             links={[
               {
-                href: paths.directive(),
+                href: paths.iap(),
                 mode: "solid",
```

Comment:
I think the link is supposed to go to `en/indigenous-it-apprentice/hire` that fits the context more as well

![image](https://github.com/user-attachments/assets/490e2a5e-e500-472a-ab19-0e9eef9f0200)

![image](https://github.com/user-attachments/assets/a8128660-04fa-41ff-928c-16b1a730ae27)

Reply:
Updated in [update path for hire an apprentice](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11436/commits/9837925cea89018d8bae063584b30403a1e5a7c2)